src/libertem/analysis/com.py

```diff
@@ -295,3 +294,22 @@ def get_parameters(self, parameters):
     @classmethod
     def get_template_helper(cls):
         return ComTemplate
+
```

Comment:
OMG it recalculates live when I change the parameters without re-running the UDF? NICE! Can I have a slider for the angle, please? 😁